cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -873,17 +950,23 @@ void esp_wifi_setup (esp_wifi_netdev_t* dev)
         return;
     }
 
+#ifndef MODULE_ESP_WIFI_AP
     /* register RX callback function */
     esp_wifi_internal_reg_rxcb(ESP_IF_WIFI_STA, _esp_wifi_rx_cb);
+#endif /* MODULE_ESP_WIFI_AP */
 
```

Comment:
Probably, this can be removed completely here. In station mode, `esp_wifi_internal_reg_rxcb` is called in event handling of `        `SYSTEM_EVENT_STA_CONNECTED` and `SYSTEM_EVENT_STA_DISCONNECTED`

```suggestion
```